mitiq/cdr/cdr_execution.py

```diff
@@ -79,10 +80,13 @@ def execute_with_CDR(
                                subsituted in the training circuits. The higher
                                this fraction the more costly the simulations,
                                but more successful the mitigation.
+        ansatz: the function to map noisy to exact data. Takes array of noisy
+                and data and parameters returning a float.
+        num_parameters: the number of paramters the ansatz takes.
```

Comment:
Let's do `ansatz` -> `fit` and `num_parameters` -> `num_fit_parameters`.

Also, include the needed signature of the `fit` and an example of calling it. It's clear to you and me, but not to someone reading this for the first time.

Reply:
@rmlarose actually I am afraid `fit` gives the idea of the action of fitting instead of "the function to be fitted".
So I am not sure about this renaming.

Reply:
whats the conclusion on this?

Reply:
Since we use `ansatz` for mitiq_curve_fit, I'm ok with this. Usually ansatz refers to circuit structure though, and `fit` or `fit_function` is more accurate imo.

Reply:
`fit_function` is good for me

Reply:
done